platform/servicepulse.include.md

```diff
@@ -1,5 +1,3 @@
-ServicePulse is a web application designed for administrators. It provides a clear, near real-time, high-level overview of how a system is currently functioning.
+ServicePulse is a web application designed for administrators. It provides a clear, near real-time, high-level overview of how a system is currently functioning. The UI also provides common failure recovery operations, such as retrying failed messages.
 
-Notifications are shown when an endpoint is offline or when a message fails. [Custom checks](/monitoring/custom-checks/) can be used to provide notifications for more specific scenarios. The ServicePulse UI provides common operations for failure recovery, such as retrying failed messages.
-
-NOTE: Further custom behaviours for message failures and other events can be acheived using [ServiceControl events](/servicecontrol/contracts.md).
+ServicePulse also has a rich, graphical view of [detailed performance metrics](https://docs.particular.net/monitoring/metrics/in-servicepulse). These are shown at the level of logical endpoints, physical instances, and even individual message types.
```

Comment:
This is the second "also" in this section. Perhaps the first one above should be removed?